### PR TITLE
Added in missing solicitor fields and fix null address scenarios

### PR DESCRIPTION
**Pull request by tchow8**
SolicitorName and SolicitorFirm were not being mapped.
Also Address fields with null values could cause an NPE when mapping the address to derived address format.
This also checks the null value of that particular field.

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/mapper/DivorceCaseToCCDMapper.java [hunk 1]
```diff
@@ -522,8 +528,11 @@ protected void mapRespondentSolicitorAddress(DivorceSession divorceSession, @Map
     @AfterMapping
     protected void mapDerivedRespondentSolicitorDetails(DivorceSession divorceSession,
                                                         @MappingTarget CoreCaseData result) {
-        if (Objects.nonNull(divorceSession.getRespondentSolicitorName())) {
-            String solicitorAddress = join(LINE_SEPARATOR,
+        if (Objects.nonNull(divorceSession.getRespondentSolicitorName())
```
Comment (qzhou-hmcts) [inline src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/mapper/DivorceCaseToCCDMapper.java]: When you have the solicitor name but not the address should you fill this data with just the solicitor name?

File: src/test/resources/fixtures/divorcetoccdmapping/divorce/null-address-fields.json [hunk 2]
```diff
@@ -0,0 +1,217 @@
+{
+    "cookie": {
```
Comment (qzhou-hmcts) [inline src/test/resources/fixtures/divorcetoccdmapping/divorce/null-address-fields.json]: Can you remove the cookie and expires?

File: src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/mapper/DivorceCaseToCCDMapper.java (hunk 1 above)
Comment (tchow8) [reply src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/mapper/DivorceCaseToCCDMapper.java]: I initially did, but Address is actually mandatory when there is a Respondent Solicitor. Don't want to store any derived data until all the required data comes though.

File: src/test/resources/fixtures/divorcetoccdmapping/divorce/null-address-fields.json (hunk 2 above)
Comment (tchow8) [reply src/test/resources/fixtures/divorcetoccdmapping/divorce/null-address-fields.json]: I'll remove them, although it shouldn't make a difference. It should be more accurate though

**Pull request merged**